src/classes/VOL_CTRL_SendBulkEmail.cls

```diff
@@ -30,10 +30,11 @@
 
 public with sharing class VOL_CTRL_SendBulkEmail {
 
-    public string strSubject { get; set; }
-    public string strMessage { get; set; }
+    public string subject { get; set; }
+    public string message { get; set; }
```

Comment:
@jjbennett  Since you're here, would you mind fixing the case on these var type declarations. I don't think we need to go any further with refactoring though.